agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -206,11 +206,10 @@ static void son_slave_watchdog(const std::string &beerocks_slave_temp_path,
     }
 }
 
-static int system_hang_test(beerocks::config_file::sConfigSlave &beerocks_slave_conf, int argc,
-                            char *argv[])
+static int system_hang_test(const beerocks::config_file::sConfigSlave &beerocks_slave_conf,
+                            int argc, char *argv[])
 {
     std::string name = std::string("system_hang_test");
-    std::ofstream versionfile;
```

Comment:
This one again is separate.

You could combine a few unused variable fixes in a single commit, if you want to reduce the number of commits.